### PR TITLE
add orderGlosses to GalleryEntry

### PR DESCRIPTION
**Pull request by Danble**
#### Relevant Issue
(prepend "closes" if issue will be closed by PR)

#### Summarize what changed in this PR (for developers)
I added the orderGlosses function to the gallery view and added one more test to see what getting the first available gloss would look like.

#### How can the changes be tested? 
Please also provide applicable links using relative paths from root (e.g. `/apatani/entries`) and reviewers can just add that onto preview urls or localhost.
/iquito/entries/gallery
https://living-dictionaries-git-fix-gallery-glosses-livingtongues.vercel.app/iquito/entries/gallery?entries_prod%5Btoggle%5D%5BhasImage%5D=true

#### Checklist before marking ready to merge
Please keep it in draft mode until these are completed:
- [x] Equal time was spent cleaning the code as writing it (Boy Scout Rule)
  - [x] Functions
    - [x] Functions that don't belong in Svelte components are extracted out into `.ts` files
    - [x] Functions are short and well named
    - [x] Concise tests are written for all functions
  - [ ] Classes (a Svelte Component is a Class)
    - [ ] Svelte components are broken down into smaller components so that each component is responsible for one thing (Single Responsibility Principle)
    - [ ] Stories/variants are written to describe use cases
  - [x] Comments are only included when absolutely necessary information that cannot be explained in code is needed

### CODE REVIEW

File: packages/site/src/lib/helpers/glosses.test.ts [hunk 1]
```diff
@@ -62,6 +62,64 @@ describe('orderGlosses', () => {
   });
 });
 
+describe('orderGlosses with multiple examples', () => {
+  const $t = (id: string) => {
```
Comment (Danble) [inline packages/site/src/lib/helpers/glosses.test.ts:66]: @jacob-8 could I use general constants in tests if I'm reusing code?

Comment (jacob-8) [reply packages/site/src/lib/helpers/glosses.test.ts:66]: I don't understand the question, can you please explain further?

File: packages/site/src/lib/helpers/glosses.test.ts [hunk 2]
```diff
@@ -62,6 +62,64 @@ describe('orderGlosses', () => {
...
+            glosses: entry.gl,
+            dictionaryGlossLanguages,
+            $t,
+          })[0]
+      )
+    ).toMatchInlineSnapshot(`
+      [
+        "apfel",
+        "plátano",
+        undefined,
```
Comment (jacob-8) [inline packages/site/src/lib/helpers/glosses.test.ts:117]: @Danble this should not be returning undefined

**Comment by jacob-8** [comment]
Will return to after #274

**Comment by jacob-8** [comment]
@Danble Can you please update this branch to be current and then I can review it?

**Pull request closed without merging**